region/brinstar/pink/Big Pink.json

```diff
@@ -1318,6 +1542,51 @@
         "A Power Bomb also works, but will clear B, Samus can just go 4->10->13."
       ]
     },
+    {
+      "link": [4, 15],
+      "name": "G-Mode Morph, Avoid Crumbles, Bomb Power Bomb Blocks",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
```

Comment:
Add a copy strat for indirect?

Reply:
It could maybe be added - it would require coming in with a jump or bomb boost, because otherwise you will fall onto the crumble.

Reply:
I'll put this one in as a fixme, because it if was a water room, I dont think it would work. (without gravity, spring, or morph)